### DISCUSSION TITLE
Publish to scoop

### DISCUSSION THREAD

**Discussion by Milo123459** [red/red#4874] [Ideas]
Publish to [scoop](https://scoop.sh) - it's a better package manager for windows

**Comment by Milo123459** [comment]
Alright then?

**Comment by hiiamboris** [comment]
This is not the place for ad spamming. If you wish red on scoop, cooperate your efforts with the team on [Gitter](https://gitter.im/red/red)

**Comment by Milo123459** [comment]
I'm not spamming ads I'm simply suggesting that it is published to scoop.

**Comment by hiiamboris** [comment]
We welcome champions if you wish to be one ;)

**Comment by Milo123459** [comment]
> We welcome champions if you wish to be one ;)

Whatever that means?

**Comment by hiiamboris** [comment]
Actually, this sounded familiar so this is what I've found: https://github.com/red/scoop-bucket
You sure Red isn't on scoop yet?

**Reply by Milo123459** [reply]
My bad. I didn't see it in the installation instructions.

**Reply by hiiamboris** [reply]
That settles it then :+1:

**Reply by dander** [reply]
I didn't see this discussion until now.

Yep, in fact, the 'stable' version of Red is in the 'main' Scoop bucket. It would be worth adding the scoop reference on https://www.red-lang.org/p/download.html though. I'm not sure who would handle that.
